src/utilities/getTokenByAddress.ts

```diff
@@ -2,14 +2,16 @@ import { Token } from 'types';
 
 import { TOKENS } from 'constants/tokens';
 
-const getTokenByAddress = (address: string) => {
+const getTokenByAddress = (address: string | null) => {
   let token: Token | undefined;
 
   Object.keys(TOKENS)
     .filter(key => Object.prototype.hasOwnProperty.call(TOKENS, key))
     .forEach(tokenId => {
       const currentToken = TOKENS[tokenId as keyof typeof TOKENS];
-      if (currentToken?.address.toLowerCase() === address.toLowerCase()) {
+      if (!address) {
+        token = TOKENS.bnb;
+      } else if (currentToken?.address.toLowerCase() === address.toLowerCase()) {
```

Comment:
You could use the areTokensEqual util here (it does exactly the same thing)

Reply:
`areTokensEqual` wouldn't be compatible with `address`, as it is a string here (maybe we want another util for these cases?)

Reply:
Sorry, I mistyped. The util I was thinking about is `areAddressesEqual`

Reply:
There you go, I was just looking at the list of utils for it, done!